contracts/test/mocks/apps/disputable/DisputableAppMock.sol

```diff
@@ -36,11 +36,15 @@ contract DisputableAppMock is DisputableAragonApp {
     function interfaceID() external pure returns (bytes4) {
         IDisputable iDisputable;
         return iDisputable.setAgreement.selector ^
-        iDisputable.onDisputableActionChallenged.selector ^
-        iDisputable.onDisputableActionAllowed.selector ^
-        iDisputable.onDisputableActionRejected.selector ^
-        iDisputable.onDisputableActionVoided.selector ^
-        iDisputable.getAgreement.selector;
+            iDisputable.onDisputableActionChallenged.selector ^
+            iDisputable.onDisputableActionAllowed.selector ^
+            iDisputable.onDisputableActionRejected.selector ^
+            iDisputable.onDisputableActionVoided.selector ^
+            iDisputable.getAgreement.selector ^
+            iDisputable.getDisputableAction.selector ^
```

Comment:
We removed this one for now, the CI is failing because of this, let's fix it and merge after it passes!

Reply:
Yes, I saw, but forgot to push last night, doing it now.